developers/weaviate/configuration/inverted-indexes.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Inverted indexes
+sidebar_position: 15
+image: og/docs/indexes.jpg
+# tags: ['configuration']
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import PyCodeV4 from '!!raw-loader!/_includes/code/howto/indexes/indexes-inverted-v4.py';
+import PyCodeV3 from '!!raw-loader!/_includes/code/howto/indexes/indexes-inverted-v3.py';
+import TSCodeV3 from '!!raw-loader!/_includes/code/howto/indexes/indexes-inverted-v3.ts';
+import TSCodeV2 from '!!raw-loader!/_includes/code/howto/indexes/indexes-inverted-v2.ts';
+
+Weaviate uses [inverted indexes](/developers/weaviate/concepts/indexing#inverted-indexes), also known as keyword indexes, to make textual and numeric searches more efficient. Weaviate provides different kinds to inverted index so you can match better match the index to your data.
+
+These indexes are normally configured on a property level:
```

Comment:
Not sure I agree with "normally" configured on a property level. Some aspects like whether to enable or disable a particular inverted index, are configurable on a property level - but others, like whether to index metadata, are configurable on a collection level.